apps/web/src/components/Test/Weights.tsx

```diff
@@ -93,8 +99,18 @@ const Weights = ({ options }: { options: ClientOption[] }) => {
           option={option}
         />
       ))}
-      <div className="flex flex-col items-end">
-        <Button onClick={onSave}>Save</Button>
+      <div className="flex justify-between">
+        {weightsSum !== 100 ? (
+          <p className="text-center text-accent-background">
```

Comment:
I think it looks better with this
```suggestion
          <p className="text-center text-xs text-accent-background">
```